src/scss/core/_utilities.scss

```diff
@@ -11,11 +11,22 @@
 @include bg-variant(".bg-gray-100", $gray-100, true);
 @include bg-variant(".bg-gray-400", $gray-400, true);
 
+// Position
+.top-30 {
```

Comment:
these are customizations of the original carousel, if original carousel has changed we need to talk with the designer, asking why

Reply:
let me double check, looks like we have carousel centered on image prev/next button for products carousel in the homepage

Reply:
updated code following same way with product carousel